### PR TITLE
🐛 Fixes in api-server for metamodeling

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

Various fixes in the `api-server` while checking meta-modeling functionality

- 🐛 Fixes `webserver`'s response in `update_node_outputs`  (reported by @wvangeit )
- 🐛 Fixes `url` fields missing in api-server's `Job` responses (reported by @wvangeit )
- 🐛 Fixes probes not rendering because it had wrong defaults  (reported by @odeimaiz )
- 🎨 Ensure that all changes in https://github.com/ITISFoundation/osparc-simcore/compare/master...wvangeit:osparc-simcore:studies_api are in place. 

These changes were split from PR https://github.com/ITISFoundation/osparc-simcore/pull/5660 to simplify review.

## Related issue/s

- Follows up PR https://github.com/ITISFoundation/osparc-simcore/pull/5656
- Fixes parts of  https://github.com/ITISFoundation/osparc-issues/issues/1291
- Fixes https://github.com/ITISFoundation/osparc-simcore/issues/5674

## How to test

- `api-server` and `webserver` tests
- draw this study in front-end
image

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by wvangeit — APPROVED** [review]
Looks good. Thanks a lot.

File: packages/models-library/src/models_library/function_services_catalog/services/probes.py [hunk 1]
```diff
@@ -1,30 +1,29 @@
...
             ],
             "contact": OM.email,
             "inputs": {
                 "in_1": {
-                    "label": f"{type_name} Probe",
-                    "description": f"Captures {type_name} values attached to it",
-                    "defaultValue": 0,
+                    "label": f"Sensed {type_name}",
+                    "description": f"Captured {type_name} value",
+                    # NOTE: no default provided to input probes
```
Comment (odeimaiz) [inline packages/models-library/src/models_library/function_services_catalog/services/probes.py:26]: 👍

**Review by odeimaiz — APPROVED** [review]
Merci

**Review by matusdrobuliak66 — APPROVED** [review]

**Review by sanderegg — APPROVED** [review]
thanks!

**Review by GitHK — APPROVED** [review]

**Pull request merged**